alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -190,6 +191,19 @@ class AlertService(
             )
         }
 
+        // Including a list of triggered clusters for cluster metrics monitors
+        var triggeredClusters: MutableList<String>? = null
+        if (result is ClusterMetricsTriggerRunResult)
```

Comment:
`composeQueryLevelAlert` method is now checking for ClusterMetricsTriggerRunResult

Reply:
why aren't we adding `it.cluster` to all the alerts irrespecitve of monitor type? even if we dont support the other monitor types it will just show local cluster id right?

Reply:
Correct. Cluster metrics monitors started out as query level monitors before we added additional monitor types when we introduced bucket level monitors. Their alerts were handled in a similar way up until this point, so I added this check in `composeQueryLevelAlert` rather than duplicate the code elsewhere.